modules/machineset-yaml-rhv.adoc

```diff
@@ -126,6 +133,13 @@ If you are using a version earlier than {rh-virtualization} 4.4.8, see link:http
 <22> Optional: AutoPinningPolicy defines the policy that automatically sets CPU and NUMA settings, including pinning to the host for this instance. Supported values: `none`, `resize_and_pin`. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Setting_NUMA_Nodes[Setting NUMA Nodes] in the _Virtual Machine Management Guide_.
 <23> Optional: Hugepages is the size in KiB for defining hugepages in a VM. Supported values: `2048` or `1048576`. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_Huge_Pages[Configuring Huge Pages] in the _Virtual Machine Management Guide_.
 <24> Optional: A list of affinity group names to be applied to the VMs. The affinity groups must exist in oVirt.
+<25> Sparse: Setting this option to `false` enables preallocation of disks. Default: `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
+<26> Format: Can be set to `cow` or `raw`. Default: `cow`. The 'cow' format is optimized for virtual machines.
```

Comment:
You could remove "Format:" from this line, because the call out numbering shows which option is being referred to. This would provide consistency with the other call outs.

Reply:
You could update "Default: `cow`" to be a full sentence. For example:

> The default is `cow`.